src/main/resources/fxml/local_server_services.fxml

```diff
@@ -1,60 +1,37 @@
+
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.control.Button?>
 <?import javafx.scene.control.ComboBox?>
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.control.TextArea?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.image.ImageView?>
 <?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.VBox?>
-<?language javascript?>
-
 <BorderPane fx:controller="com.esri.samples.localserver.local_server_services.LocalServerServicesController"
-           xmlns:fx="http://javafx.com/fxml">
-  <!-- Top Section -->
+            xmlns:fx="http://javafx.com/fxml">
+  <!-- Select and Start Service -->
   <top>
-    <HBox spacing="10">
-      <padding><Insets top="10" right="10" bottom="5" left="10"/></padding>
-    
-      <VBox spacing="3"> 
-        <Button fx:id="btnStartServer" text="Start Local Server" onAction="#handleStartLocalServer" minWidth="200"/>
-        <Button text="Stop Local Server"  onAction="#handleStopLocalServer" disable="${!btnStartServer.disable}"
-                minWidth="200"/>
-      </VBox>
-    
-      <VBox HBox.hgrow="ALWAYS" spacing="3"> 
-        <HBox>
-          <ComboBox fx:id="serviceOptions" minWidth="200" maxWidth="Infinity" disable="${!btnStartServer.disable}" onAction="#handleSelectServiceOption">
-          
-            <value>
-              <String fx:value="Map Service"/>
-            </value>
-          
-            <items>
-              <FXCollections fx:factory="observableArrayList">
-                <String fx:value="Map Service"/>
-                <String fx:value="Feature Service"/>
-                <String fx:value="Geoprocessing Service"/>
-              </FXCollections>
-            </items>
-          </ComboBox>
-          <Button text="Select package" disable="${!btnStartServer.disable}" minWidth="100" onAction="#handleSelectPackage"/>
-          <Label fx:id="selectedPackageLabel"/>
-        </HBox>
-        
-        <HBox spacing="3">
-          <Button HBox.hgrow="ALWAYS" text="Start Service" maxWidth="Infinity"
-            onAction="#handleStartSelectedService" disable="${!btnStartServer.disable}"/>
-          <Button fx:id="btnStopService" HBox.hgrow="ALWAYS" text="Stop Service" maxWidth="Infinity"
-            onAction="#handleStopSelectedService"/>
-        </HBox>
- 
-      </VBox>
-    </HBox>
+    <GridPane hgap="5" vgap="5">
+      <padding><Insets left="10" top="10" right="10" bottom="10"/></padding>
+      <TextField fx:id="packagePath" editable="false" GridPane.rowIndex="0" GridPane.columnIndex="0" GridPane.hgrow="ALWAYS"/>
+      <Button onAction="#openPackage" text="Open" GridPane.rowIndex="0" GridPane.columnIndex="1">
+        <graphic>
+          <ImageView>
+            <Image url="/icons/files-48.png" requestedHeight="16" preserveRatio="true" smooth="true"/>
```

Comment:
I couldn't get this sample to run without commenting this out. Couldn't find location to icon